resource_detectors/.yardopts

```diff
@@ -0,0 +1,8 @@
+--no-private
+--title=OpenTelemetry Resource Detectors
+--markup=markdown
+--main=README.md
+./lib/opentelemetry/resource/detectors/**/*.rb
+./lib/opentelemetry/resource/detectors.rb
+-
+README.md
```

Comment:
Following the great comment left here :) https://github.com/open-telemetry/opentelemetry-ruby/pull/326#issuecomment-678486553